exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/HashJoinProbe.java

```diff
@@ -43,11 +39,7 @@
     PROBE_PROJECT, PROJECT_RIGHT, DONE
   }
 
-  void setupHashJoinProbe(FragmentContext context, VectorContainer buildBatch, RecordBatch probeBatch,
-                          int probeRecordCount, HashJoinBatch outgoing, HashTable hashTable, HashJoinHelper hjHelper,
-                          JoinRelType joinRelType, RecordBatch.IterOutcome leftStartState);
-  void doSetup(FragmentContext context, VectorContainer buildBatch, RecordBatch probeBatch, RecordBatch outgoing);
-  int  probeAndProject() throws SchemaChangeException, ClassTransformationException, IOException;
-  void projectBuildRecord(int buildIndex, int outIndex);
-  void projectProbeRecord(int probeIndex, int outIndex);
+  void setupHashJoinProbe(RecordBatch probeBatch, HashJoinBatch outgoing, JoinRelType joinRelType, RecordBatch.IterOutcome leftStartState, HashPartition[] partitions, int cycleNum, VectorContainer container, HashJoinBatch.HJSpilledPartition[] spilledInners, boolean buildSideIsEmpty, int numPartitions, int rightHVColPosition);
```

Comment:
Line too long.. can you split this ?  Do we have a check style for max line length ?